static/js/metrics/graphs/activeDevices.js

```diff
@@ -0,0 +1,162 @@
+/* globals bb */
+
+import { formatAxis, formatXAxisTickLabels, formatYAxisTickLabels } from '../axis';
+import debounce from '../../libs/debounce';
+import { snapcraftGraphTooltip, positionTooltip } from '../tooltips';
+import { COLORS } from '../config';
+
+function showGraph(el) {
+  formatAxis(el);
+  el.style.opacity = 1;
+}
+
+function findStart(data) {
+  for (let i = 0, ii = data.length; i < ii; i += 1) {
```

Comment:
`for (let i = 0; i < data.length; i += 1) {`

Reply:
the ii caches the `.length` so it doesn't need to be checked each iteration.